### PR TITLE
adscript: add Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by EmilySeville7cfg — APPROVED** [review]
No fixes found, approved.
Thanks for a contribution! ❤️ 🎄

File: pages.es/common/adscript.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# adscript
+
+> Compilador de archivos Adscript.
+> Más información: <https://github.com/Amplus2/Adscript>.
+
+- Compilar un archivo en un archivo objeto:
+
+`adscript --output {{path/to/file.o}} {{path/to/input_file.adscript}}`
```
Comment (EmilySeville7cfg) [inline pages.es/common/adscript.md:8]:
```suggestion
- Compilar un archivo en un archivo objeto:

`adscript --output {{ruta/al/archivo.o}} {{ruta/al/archivo_de_entrada.adscript}}`
```
Optional fix: placeholders can be translated.

**Review by EmilySeville7cfg — APPROVED** [review]
No mandatory fixes found, approved.
Thanks for a contribution! 🎄

**Review by kbdharun — APPROVED** [review]
LGTM, after @EmilySeville7cfg's suggestions.

**Review by pixelcmtd — CHANGES_REQUESTED** [review]
What @EmilySeville7cfg said, the rest LGTM.

**Review by pixelcmtd — APPROVED** [review]

**Comment by kant** [comment]
@kbdharun @pixelcmtd @EmilySeville7cfg Best brand new year!

**Comment by kbdharun** [comment]
> @kbdharun @pixelcmtd @EmilySeville7cfg Best brand new year!

Happy new year 🎉

**Comment by pixelcmtd** [comment]
It's 22:07, but still happy new year y'all

**Comment by EmilySeville7cfg** [comment]
Thanks! ❤️

**Pull request merged**